tasks/hyrax-metadata-updates/schemas/config.json

```diff
@@ -6,29 +6,43 @@
     "cmr"
   ],
   "properties": {
-    "cmr": {
+    "hyrax": {
       "type": "object",
-      "description": "Credentials needed to perform CMR searches and metadata validation",
-      "required": [
-        "provider",
-        "clientId",
-        "username",
-        "passwordSecretName"
-      ],
+      "description": "Hyrax specific configuration",
+      "required": [],
       "properties": {
-        "provider": {
-          "type": "string"
-        },
-        "clientId": {
-          "type": "string"
-        },
-        "username": {
-          "type": "string"
-        },
-        "passwordSecretName": {
-          "type": "string"
+        "formats": {
+          "type": "array",
```

Comment:
Might be nice to have a `description` here as well

Reply:
Committed